client/packages/system/src/Encounter/DetailView/Toolbar.tsx

```diff
@@ -43,8 +44,25 @@ export const Toolbar: FC<ToolbarProps> = ({ onChange }) => {
         flexDirection="row"
         display="flex"
         flex={1}
-        alignItems="flex-end"
+        alignItems="center"
       >
+        <Grid
+          item
+          sx={{
+            alignItems: 'center',
+            backgroundColor: 'background.menu',
+            borderRadius: '50%',
+            display: 'flex',
+            height: '100px',
+            justifyContent: 'center',
+            marginRight: 2,
+            width: '100px',
+          }}
+        >
+          <Box>
+            <UserIcon fontSize="large" style={{ flex: 1 }} />
+          </Box>
+        </Grid>
```

Comment:
My only problem with this icon is that it now looks like one of those "placeholder" icons you often get on social media etc when someone hasn't uploaded their actual profile picture yet. So it makes this look like there should be a photo of the patient there but we just haven't added it yet.

Unfortunately I don't really have a better suggestion. Maybe an icon more specific to "Encounter", like one of these:
![Screenshot 2023-01-09 at 3 22 46 PM](https://user-images.githubusercontent.com/5456533/211232036-b3f3865b-8354-4f82-8fc2-5086deadf095.png)

And maybe in orange with no grey circle? (it's the grey circle that gives it the "placholder" feel, I think)

If that's no better, then maybe just leave it as you've got it until Anisha can weigh in.

(I actually think it's fine with nothing, as it seems more consistent with the other pages, but I guess it was a specific request)

Reply:
ditto! I thought it looked a bit placeholder-y and also didn't feel that it was needed.. but can see the point, that a distinction between the encounter and patient areas is useful. Will leave it for now.. and get feedback from Anisha